scripts/src/chartprreview/chartprreview.py

```diff
@@ -221,6 +222,13 @@ def check_report_success(directory, report_path, version):
         write_error_log(directory, msg)
         sys.exit(1)
 
+    if "charts.openshift.io/certifiedOpenShiftVersions" in annotations:
+        full_version = annotations["charts.openshift.io/certifiedOpenShiftVersions"]
+        if not semver.VersionInfo.isvalid(full_version):
+            msg = f"[ERROR] No matching OpenShift version found: {full_version}"
```

Comment:
IMHO, the message could be improved, because we land here if the `full_version` is not a valid sem version.

Reply:
Done. 908e8605354d7c73c1171c77f37b9ffeee336197